client/src/app/TabsProvider.js

```diff
@@ -88,6 +96,51 @@ export default class TabsProvider {
         },
         getNewFileButton() {
           return null;
+        },
+      },
```

Comment:
A suggestion: We could write a test that verifies the correct menu item order is show.

We could also attach the priority to the menu item _only_, as this is the place where it matters at the moment, as far as I understand?

Reply:
+1 for the test case

The priority is _not_ for the menu item, but for which provider will be tried to open the file first (https://github.com/camunda/camunda-modeler/pull/2167/files#diff-a8ee53f63c47900983b24616f87f9b04023e1b48be19fd3ee726d0138276b811R493).

I tried to use it for the menu item only, which was more difficult as expected (https://github.com/camunda/camunda-modeler/pull/2167#issue-596577291). That's why I didn't do so.

Reply:
Fine as long as we have a test case that asserts the correct behavior.

Reply:
> The priority is not for the menu item, but for which provider will be tried to open the file first (https://github.com/camunda/camunda-modeler/pull/2167/files#diff-a8ee53f63c47900983b24616f87f9b04023e1b48be19fd3ee726d0138276b811R493).

:+1: on this one.

:+1: on not having a test case, too. A proper test case would require a substatntial amount of work and getting it _stable_ would require introduction of new APIs, too. Let's not do that for now :wink:

Reply:
❤️